src/action/mod.rs

```diff
@@ -390,6 +407,11 @@ pub enum ActionErrorKind {
         "`{0}` exists with different content than planned, consider removing it with `rm {0}`"
     )]
     DifferentContent(std::path::PathBuf),
+    /// The path already exists with a different line content that expected
+    #[error(
+        "`{0}` exists with different line content than planned, the line may have changed since read"
```

Comment:
Maybe we could also work the `Position` into the error message, like `at {position.line}:{position.col}` or whatever `Position` has?

Reply:
I thought I did, whoops!